isis/src/docsys/documents/DeveloperAddFeature/DeveloperAddFeature.xml

```diff
@@ -0,0 +1,187 @@
+            starting to write your first application.
+          </p>
+          <p>
+            There are over 400 <a href="https://isis.astrogeology.usgs.gov/Application/index.html"ISIS applications</a>.
+            We suggest you get familiar with as many of these as possible, so you don't duplicate a feature that has already 
+            been implemented, and to use an existing application as a template for your new feature.
+          </p>
+          <p>
+            ISIS applications are linked against many external libraries. We encourage you to become familiar with these 
+            external libraries. One place to start is the environment.yml file located at the root of the ISIS source
+            code tree. 
+          </p>
+
+        <h2><a name="Getting the ISIS source code">Getting the ISIS source code</a></h2>
+          <p>
+            All of the ISIS source code is available on <a href="https://github.com/USGS-Astrogeology/ISIS3">GitHub</a>.
+            This GitHub source code repository is the primary Git repository for ISIS. If you are unfamiliar with
+            Git and/or GitHub you should go through the GitHub <a href="https://docs.github.com/en/get-started">getting started</a>
+            pages before you start writing code.
+          </p>
+
+          <h3>Steps to get ISIS source code on your local machine</h3>
+            <ol>
+              <li>Setup a GitHub account</li>
+              <li>Fork the ISIS repository</li>
+              <li>Clone your fork to a local repository</li>
+            </ol>
+
+          <h2><a name="Setting up an environment to compile and link ISIS">Setting up an environment to compile and link ISIS</a></h2>
+            <p>
+              The ISIS package is written in C++. In order to run and test your new feature, you will need to compile the source code and
+              link it against the ISIS library and the external libraries. Follow the instructions on the ISIS GitHub Wiki for 
+              <a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake"> building ISIS</a>.
+            </p>
+            <p>
+              Depending on how your computer is setup, you may also need to install:
+            </p>
+            <ul>
+              <li> Compiler (Mac OS Xcode, Linux GNU C++)</li>
+              <li> Editor (Vim, Nano, Atom, ...</li>
+            </ul>
+
+            <p>
+              Once you have a complete environment set up for ISIS development, we suggest you 
+              <a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#building-isis3">compile and build</a> 
+              all of ISIS. This step will make sure sure you have everything setup and ready to go.
+            </p>
+
+          <h2><a name="Starting a conversation with other ISIS developers">Starting a conversation with other ISIS developers</a></h2>
```

Comment:
add a link to the contributing document that outlines expectations of contributions and interactions https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md

Reply:
Placed nearer the top